app/models/qrda_file.rb

```diff
@@ -142,8 +142,8 @@ def validators
       elsif program_type == "none"
         if doc_type == "cat3_r1"
           HealthDataStandards::Validate::Cat3
-        elsif doc_type == "cat3_r11"
-          HealthDataStandards::Validate::Cat3R11
+        elsif doc_type == "cat3_r2"
+          HealthDataStandards::Validate::Cat3R2

```

Comment:
This code is never going to be executed. The `when "cat3_r1", "cat3_r11"` above keeps `"cat3_r2"` values of `doc_type` out of this if statement.